services/migrations/gitea_uploader.go

```diff
@@ -760,10 +760,15 @@ func (g *GiteaLocalUploader) newPullRequest(pr *base.PullRequest) (*issues_model
 		pr.Updated = pr.Created
 	}
 
+	prTitle := pr.Title
+	if pr.IsDraft {
+		prTitle = fmt.Sprintf("%s%s", setting.Repository.PullRequest.WorkInProgressPrefixes[0], pr.Title)
```

Comment:
Erm… I feel like this will lead to duplicate prefixes.

Reply:
And, at least, there should be a space

Reply:
I added a check for the WIP prefix in `f15a17cdc47cc6d8d313435b946f42e07fe902c0`. Please help review.